src/components/What.tsx

```diff
@@ -1,7 +1,35 @@
 import React from "react";
 
 const What = () => {
-  return <div>What</div>;
+  return (
```

Comment:
look at about section pull request since the structure to figure out the position since it is basically the same but inverse